init.js

```diff
@@ -15,12 +15,86 @@ $(document).ready(function () {
     // load the highlight.min.js script this way to avoid Uncaught SyntaxError: unterminated regular expression literal
     $.cachedScript('https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.5.1/highlight.min.js')
         .done(function (script, textStatus) {
+            $.cachedScript('https://cdnjs.cloudflare.com/ajax/libs/highlightjs-line-numbers.js/2.8.0/highlightjs-line-numbers.min.js')
```

Comment:
This is my main concern about your PR:
Highlight.js doesn't manage line numbers to keep simple, and I have the same wish for this extension.
But as your already made the work, I would suggest you to create 2 separate PRs for this concern:

- one for the line numbers
- another one for the highlight of specific lines

Reply:
On second thought, I think you should create a separate extension to display the line numbers.